src/extension/options-page/index.tsx

```diff
@@ -45,7 +45,7 @@ const useStyles = makeStyles((theme) => {
         wrapper: {
             '--monospace': 'SFMono-Regular, Consolas, "Liberation Mono", Menlo, monospace',
 
-            '--drawerHeader': dark ? '#121212' : theme.palette.primary.dark,
+            '--drawerHeader': dark ? '#121212' : theme.palette.primary.main,
```

Comment:
Hmm why

Reply:
It's too dark as the background color of the drawer.